### PR TITLE
Fixed deprecations this.property-fallback, with, and positional linkt…

### PR DESCRIPTION
**Pull request by bgantzler**
…o parms

### CODE REVIEW

File: tests/acceptance/sandbox/api/guides-test.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ module('Acceptance | Sandbox | API | Guides', function(hooks) {
 
   test('page index works', async function(assert) {
     await visit('/sandbox');
-
+debugger;
```
Comment (rwwagner90) [inline tests/acceptance/sandbox/api/guides-test.js:14]:
```suggestion
```

File: index.js [hunk 2]
```diff
@@ -13,6 +13,10 @@ const LATEST_VERSION_NAME = '-latest';
 const styleDir = path.join(__dirname, 'addon', 'styles');
 
 module.exports = {
+  isDevelopingAddon: function () {
+    return true;
+  },
+
```
Comment (rwwagner90) [inline index.js:19]:
```suggestion
```

Was this needed for something?

**Review by rwwagner90 — CHANGES_REQUESTED** [review]
Thanks for the PR! Was just about to address this myself.

**Review by rwwagner90 — APPROVED** [review]

**Comment by cah-briangantzler** [comment]
OMG, must have been doing too many things. Sorry for leaving the debugger and developing addon in. I know better :(

**Comment by rwwagner90** [comment]
No worries @cah-briangantzler! I really appreciate the help 😃

**Pull request merged**